samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -153,7 +171,8 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
-When building the sample, you must provide the Fast Pair Model ID (:c:macro:`FP_MODEL_ID`) and the Fast Pair Anti Spoofing Key (:c:macro:`FP_ANTI_SPOOFING_KEY`) as CMake options.
+When building the sample, you can provide the Fast Pair Model ID (:c:macro:`FP_MODEL_ID`) and the Fast Pair Anti-Spoofing Key (:c:macro:`FP_ANTI_SPOOFING_KEY`) as CMake options.
+If the data is not provided, the default provisioning data, obtained for the "NCS Fast Pair demo", the input device debug Fast Pair provider, is used.
```

Comment:
```suggestion
If the data is not provided, the sample uses the default provisioning data obtained for the "NCS Fast Pair demo" (the input device debug Fast Pair provider).
```

@alstrzebonski Tired to rephrase this part a bit, please let me know in case it affects the meaning

Reply:
It's okay. Sounds better.